packages/app-bridge/src/tests/DocumentApiDummy.ts

```diff
@@ -44,6 +44,7 @@ export class DocumentApiDummy {
             change_title: null,
             targets: [],
             token: 'a-dummy-token',
+            permanent_link: '/r/document-1234',
```

Comment:
```suggestion
            permanent_link: `/r/document-${id}`,
```
Same for the other one